cJSON.c

```diff
@@ -117,12 +120,11 @@ CJSON_PUBLIC(double) cJSON_GetNumberValue(const cJSON * const item)
     #error cJSON.h and cJSON.c have different versions. Make sure that both have the same.
 #endif
 
+#define VERSION_NUMBER (XSTR(CJSON_VERSION_MAJOR) "." XSTR(cJSON_VERSION_MINOR) "." XSTR(cJSON_VERSION_PATCH"))
```

Comment:
I already mentioned this is the other comment, but there's no need for this intermediate macro. Also, `VERSION_NUMBER` isn't very descriptive of the fact that it expands to a string literal.